inventory/group_vars/k8s-cluster.yml

```diff
@@ -54,6 +54,19 @@ kube_users:
     pass: "{{kube_api_pwd}}"
     role: admin
 
+
+## Variables for OpenID Connect Configuration https://kubernetes.io/docs/admin/authentication/
+## To use OpenID you have to deploy additional an OpenID Provider (e.g Dex, Keycloack, ...)
+# kube_oidc_auth: false  is defined and
```

Comment:
I think you meant to write `kube_oidc:auth: false` and should leave out `is defined and`